### DISCUSSION TITLE
I amworking on Deep learning Pre-trained Model And developing a api using fastapi i am working on this project since 30 day always get "ImportError"

### DISCUSSION THREAD

**Discussion by Kai-tav** [fastapi/fastapi#11361] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import APIRouter, UploadFile, File
from model import RealESRGAN
import torch  # Import from models.py
from utils.image_utils import load_image, resize_image, image_to_tensor
```

### Description

 **File "/Path/Model-F/api/routes/api_routes.py", line 2, in <module>
    from model import RealESRGAN
ModuleNotFoundError: No module named 'model'**

### Operating System

macOS

### Operating System Details

**Mac mini M2 Apple Sillicon
Ram - 8 GB
Storage - 256** 

### FastAPI Version

__version__ = "0.110.0"

### Pydantic Version

 VERSION = '2.6.4'

### Python Version

Python 3.12.2

### Additional Context

**I am trying to create API to use the deep learning model's functionality using fastapi i have tried every possible way but have been stuck at "ImportError" since last 30 days and the pr**oject deadline is today id anyone can help!!

**Comment by PhysicallyActive** [comment]
This does not appear to be a FastAPI issue but a python-path issue. Can you print out your directory structure? For instance,
```
root/
    models.py
    utils/
        image_utils.py
        # Some more files?
```

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
